src/python/sdk/examples/trade.py

```diff
@@ -0,0 +1,533 @@
+        else:
+            return url_in
+    else:
+        return url
+
+
+ECONIA_ADDR = (
+    get_econia_address()
+)  # See https://econia.dev/ for up-to-date per-chain addresses
+FAUCET_ADDR = get_faucet_address()  # See (and deploy): /econia/src/move/faucet
+COIN_TYPE_ETH = f"{FAUCET_ADDR}::test_eth::TestETH"
+COIN_TYPE_USDC = f"{FAUCET_ADDR}::test_usdc::TestUSDC"
+COIN_TYPE_APT = "0x1::aptos_coin::AptosCoin"
+NODE_URL = get_aptos_node_url()
+FAUCET_URL = get_aptos_faucet_url()
+
+
+def start():
+    rest_client = RestClient(NODE_URL)
+    faucet_client = FaucetClient(FAUCET_URL, rest_client)
+    viewer = EconiaViewer(NODE_URL, ECONIA_ADDR)
+    print(f"Econia address (from view function): {get_address(viewer)}")
+
+    input("\nPress enter to initialize (or obtain) the market.")
+    market_id = setup_market(faucet_client, viewer)
+
+    input("\nPress enter to set-up an Account A with funds.")
+    account_A = setup_new_account(viewer, faucet_client, market_id)
+    print(f"Account A was set-up: {account_A.account_address}")
+
+    input("\nPress enter to place limit orders with Account A.")
+    # Bid to purchase 1 whole ETH at a price of 1 whole USDC per lot!
+    # = $1000/ETH since there are 1000 lots in a whole ETH & 1 tick = 0.001 USDC
+    buy_base_lots = 1 * (10**3)
+    buy_ticks_per_lot = 1 * (10**3)
+    place_limit_order(
+        Side.BID, account_A, market_id, buy_base_lots, buy_ticks_per_lot
+    )
+    events = get_place_limit_order_events(
+        viewer, account_A.account_address, market_id, 0
+    )
+    report_place_limit_order_event(
+        list(filter(lambda ev: ev["data"]["side"] == Side.BID, events))[0]
+    )
+    # Ask to sell 1 whole ETH at a price of 2 whole USDC per lot!
+    # = $2000/ETH since there are 1000 lots in a whole ETH & 1 tick = 0.001 USDC
+    sell_base_lots = 1 * (10**3)
+    sell_ticks_per_lot = 2 * (10**3)
+    place_limit_order(
+        Side.ASK, account_A, market_id, sell_base_lots, sell_ticks_per_lot
+    )
+    events = get_place_limit_order_events(
+        viewer, account_A.account_address, market_id, 0
+    )
+    report_place_limit_order_event(
+        list(filter(lambda ev: ev["data"]["side"] == Side.ASK, events))[0]
+    )
+    print(f"Account A has finished placing limit orders.")
+    fills = get_fill_events(viewer, account_A.account_address, market_id, 0)
+    filled_size = len(fills)
+    if filled_size == 0:
+        print("  * There were no limit orders filled by any orders placed.")
+    else:
+        print(
+            f"  * There were {filled_size} limit orders filled by the orders placed."
+        )
+
+    report_best_price_levels(viewer, market_id)
+
+    input("\nPress enter to set-up and Account B with funds.")
+    account_B = setup_new_account(viewer, faucet_client, market_id)
+    print(f"Account B was set-up: {account_B.account_address}")
+
+    input("\nPress enter to place market orders (bid and ask) with Account B.")
```

Comment:
Can you change bid and ask here to buy and sell, since market orders use those terms?

Reply:
will do